libs/product/driver/magento/src/queries/fragments/product.ts

```diff
@@ -48,11 +28,24 @@ export const magentoProductFragment = gql`
 		description {
 			html
 		}
+    upsell_products {
+      ...magentoProductPreview
+      ...magentoBundledProduct
+      ...magentoSimpleProduct
```

Comment:
I don't think we want to spread the Simple product fragment here. Kind of defeats the purpose of having the preview fragment, right?

Reply:
all the simple fragment does is add the name and AFAIK we do want the name. or do you mean it should be spread into the preview fragment?

Reply:
oh whoops. I keep mixing up the differences between `DaffProduct` and `MagentoProduct`. I was thinking that `MagentoSimpleProduct` was the `MagentoProduct` (like it is in daffodil models). lgtm